### PR TITLE
PlaidML test support

### PR DESCRIPTION
**Pull request by earhart**

### CODE REVIEW

**Comment by tsocha** [comment]
maybe we should use PR #400 ?

File: conftest.py [hunk 1]
```diff
@@ -22,10 +22,15 @@ def pytest_addoption(parser):
     parser.addoption('--backend', default='CPU',
                      choices=['INTERPRETER', 'CPU', 'GPU', 'NNP', 'PlaidML', 'INTELGPU'],
                      help='Select from available backends')
+    parser.addoption('--config', default=None,
```
[the remark this answers is not in the document]
Comment (earhart) [reply conftest.py]: Done

File: ngraph_onnx/onnx_importer/backend.py [hunk 2]
```diff
@@ -53,6 +53,7 @@ class NgraphBackend(Backend):
         ('INTERPRETER', 'CPU'),
         ('ARGON', 'CPU'),
         ('NNP', 'CPU'),
+        ('PLAIDML', 'GPU'),
```
[the remark this answers is not in the document]
Comment (earhart) [reply ngraph_onnx/onnx_importer/backend.py]: Done

**Comment by earhart** [comment]
There's definitely some overlap with #400 - I think it'd be best to land that one, and then rebase and land this one, manually fixing up the conflicts.

**Comment by earhart** [comment]
No longer needed

**Pull request closed without merging**